release-doc-in-process/all_vf.TTL

```diff
@@ -30,6 +30,23 @@ vf:AgentRelationshipRole  a  owl:Class ;
         rdfs:subClassOf   rdf:Property ;
         rdfs:comment   "A relationship role defining the kind of association one agent can have with another." .
 
+vf:Recipe a               owl:Class ;
+        rdfs:label         "vf:Recipe" ;
+        rdfs:comment       "Recipes contain all the info required to create or improve or move a resource." .
+
+vf:ResourceType a         owl:Class ;
```

Comment:
Why would we need it? We already have `vf:classifiedAs` to position resources in taxonomy. I think `vf:RecipeFlow` can also reference the same taxonomy item, just as intents and commitments will often just use `vf:classifiedAs` rather than reference concrete resource.

Reply:
Because we have some properties that are useful there as part of a recipe.  The ResourceType or RecipeResource can reference one or more classifications itself to cover that need.

Reply:
Looking at this PR I only see `vf:isSubstitutable`

Reply:
isSubstitutable is enough imo... but also note, name, image go there... and I guess I didn't get some other configuration things done, it is useful to know if it is work, currency, etc. for behavior but I don't have that well defined and am not sure it should be in VF, although they would be good extensions.... also thinking about some of the standard identifiers that people could extend it with like GTIN, etc. that identify very specific resource types (could use gr vocabulary for those)

And we need something that a resource only has one of, which is not the case for classification.

Reply:
Ok, let's say we have `vf:ResourceSpecification`. I think we'll need to have two distinct properties that flows can use:
* a Flow - `vf:resourceClassifiedAs` -> a taxonomy item
* a Flow - `vf:resourceConformsTo` -> a Resource Specification

While data would in most cases also have *a Resource Specification - `vf:classifiedAs` -> a taxonomy  item. I would prefer that to get the classification we don't have to sometimes get it directly from the flow and sometimes go through the resource specification. Systems which rely on Resource Specification can always just copy over the taxonomy reference to every flow referencing that specification.

Reply:
> I would prefer that to get the classification we don't have to sometimes get it directly from the flow and sometimes go through the resource specification. Systems which rely on Resource Specification can always just copy over the taxonomy reference to every flow referencing that specification.

Makes sense to me.  But we can't define that, right?  It has totally open range and domain.  We can consider tightening that up if you like.

Also, does that mean we want to require a resource to have a resource specification?

Reply:
I don't think we can use `vf:classifiedAs` directly from the Flow. We can do it from a Resource, Process and possibly other constructs. In a Flow we don't reference taxonomy that categorizes the flow itself but the resource that this flow affects. For service flows using `vf:classifiedAs` might make sense since the flow itself seems to represent service. Though sometimes the process that outputs a service would have some `vf:classifiedAs` and not necessary the flow itself.